### PR TITLE
css-positioning - Kevin Lay

### PR DESCRIPTION
**Pull request by laykevin**
image
image
css-fixed-position11

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  top: 160px;
+  width: 300px;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+  height: 300px;
+
```
Comment (shawnkost) [inline css-positioning/absolute.css:42]: Your `.overlay` should have `position: absolute`

File: css-positioning/relative.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
```
Comment (shawnkost) [inline css-positioning/relative.css:10]: Add `position: relative` here

File: css-positioning/relative.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
```
Comment (shawnkost) [inline css-positioning/relative.css:15]: Remove this `position: relative`

File: css-positioning/relative.css [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  top: 20px;
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
```
Comment (shawnkost) [inline css-positioning/relative.css:21]: Remove this `position: relative`

File: css-positioning/relative.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  top: 40px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
```
Comment (shawnkost) [inline css-positioning/relative.css:27]: Remove this `position: relative`

File: css-positioning/relative.css [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+  top: 75px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
```
Comment (shawnkost) [inline css-positioning/relative.css:33]: Remove this `position: relative`

File: css-positioning/relative.css [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  top: 150px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
```
Comment (shawnkost) [inline css-positioning/relative.css:39]: Remove this `position: relative`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Hey Kevin! Some small changes requested below! Let me know if you have any questions!

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**